core/controllers/admin.py

```diff
@@ -134,6 +136,13 @@ def post(self):
             elif self.payload.get('action') == 'reload_collection':
                 collection_id = self.payload.get('collection_id')
                 self._reload_collection(collection_id)
+            elif self.payload.get('action') == 'generate_dummy_explorations':
+                num_dummy_exps_to_generate = self.payload.get(
+                    'num_dummy_exps_to_generate')
+                num_dummy_exps_to_publish = self.payload.get(
+                    'num_dummy_exps_to_publish')
```

Comment:
You do need to validate the args. Check that they're numbers and that one is less than or equal to the other.

In general, the backend should never trust the frontend. The frontend can be spoofed.